app/liquid_tags/glitch_tag.rb

```diff
@@ -3,6 +3,7 @@ class GlitchTag < LiquidTagBase
 
   PARTIAL = "liquids/glitch".freeze
   ID_REGEXP = /\A[a-zA-Z0-9\-]{1,110}\z/.freeze
+  TILDE_PREFIX_REGEXP = /\A~*/.freeze
```

Comment:
to be 100% correct, this matches "nothing at the beginning of a string", so basically anything you pass. That's because you use the `*`. As you then replace the ID with nothing is not a big deal but in theory this should simply be:

`\A~` which means "begins with tilde"